t/admin/token.t

```diff
@@ -177,3 +177,21 @@ PUT /apisix/admin/plugins/reload?api_key=4054f7cf07e344346cd3f287985e76a2
 --- request
 GET /apisix/admin/routes??api_key=4054f7cf07e344346cd3f287985e76a2
 --- error_code: 401
+
+
+
+=== TEST 10: access without api key
+--- request
+GET /apisix/admin/routes
+--- error_code: 401
+
+
+
+=== TEST 11: access without api key, but APISIX_BYPASS_ADMIN_API_AUTH=true
+--- main_config
+env APISIX_BYPASS_ADMIN_API_AUTH=true;
```

Comment:
need to add test cases for `APISIX_BYPASS_ADMIN_API_AUTH` is not `true`

Reply:
and if we set `APISIX_BYPASS_ADMIN_API_AUTH` to `abc`, what will happen?

Reply:
do we have a schema check for these configures?

Reply:
> and if we set `APISIX_BYPASS_ADMIN_API_AUTH` to `abc`, what will happen?

All values that are not true will be considered false.

Reply:
it's too hacky. We need a configure checker(if apisix not has) to check schema and values.

Reply:
> need to add test cases for `APISIX_BYPASS_ADMIN_API_AUTH` is not `true`

Done.

Reply:
> This variable is currently a reserved environment variable. It is not a configuration item in the configuration file (config.yaml).

can we move it to config.yaml? near by the token?

Reply:
can we set it on both sides?

Reply:
Good. we can write this to docs.